content-repo/gendocs_test.py

```diff
@@ -229,6 +229,13 @@ def test_get_deprecated_data():
     assert "Add information" not in res
 
 
+def test_get_fromversion_data():
+    res = get_fromversion_data({'fromversion': '5.5.0'})
+    assert 'Supported Cortex XSOAR versions: 5.5.0 and later.' in res
+    res = get_fromversion_data({'fromversion': '5.0.0'})
+    assert 'Supported Cortex XSOAR versions: 5.0.0 and later.' not in res
```

Comment:
Better to assert that the result is an empty string.

Reply:
done